docker/Dockerfile

```diff
@@ -46,6 +47,7 @@ RUN chmod o+r /usr/local/bundle/gems/robots-*/lib/robots.rb
 RUN adduser -g msfconsole -D $MSF_USER
 
 RUN /usr/sbin/setcap cap_net_raw,cap_net_bind_service=+eip $(which ruby)
+RUN /usr/sbin/setcap cap_net_raw,cap_net_bind_service=+eip /usr/bin/nmap
```

Comment:
As I mentioned in the PR description (sorry for my poor english tho) the problem is that `cap_net_admin` should be added to the whole image first, because docker engine do not grant this cap to an image by default. So if we really want it and accept the security implications `cap_net_admin` introduces, I'll add it to both `docker-compose.yml`s and this line too.